src/Symfony/Component/Notifier/Bridge/Smsc/composer.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/smsc-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony SMSC Notifier Bridge",
+    "keywords": ["sms", "smsc", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0|^6.0",
```

Im not sure, but I think it make sense to do

```suggestion
        "symfony/http-client": "^4.4|^5.2|^6.0",
```

Ie, only supported versions.

---

(I know this is copied from another bridge)

---

Make sense. Done